browser-sign-in/build.gradle

```diff
@@ -7,6 +7,7 @@ buildscript {
     dependencies {
         classpath libs.gradle
         classpath libs.kotlin.gradle.plugin
+        classpath libs.android.navigation.safe.args
```

Comment:
It doesn't look like you're using this plugin. Can it be removed?

Reply:
I am using it in navigate methods in LoginFragment and UserDashboardFragment